great_expectations/data_context/types/base.py

```diff
@@ -256,6 +256,9 @@ class Meta:
 
     credentials = fields.Raw(allow_none=True)
     spark_context = fields.Raw(allow_none=True)
+    spark_config = fields.Dict(
+        keys=fields.Str(), values=fields.Str(), allow_none=True
+    )
```

Comment:
Hi @ryanaustincarlson !  Thank you for catching this and proposing a fix.  I agree with the approach.  One question: Shouldn't the `spark_context` section be included as part of `spark_config`?  For example:
```
                  spark_config:
                    spark_context:
                      spark.master: local[*]
```

If so, then we would delete the line `spark_context = fields.Raw(allow_none=True)` and use your fix to allow the `spark_context` property and any other Spark related configuration parameters.

Thank you!

Reply:
@alexsherstinsky  This sounds reasonable, but I noticed that there's a nearly-identical PR from @EricSteg that's way more complete (includes updates to the docs)
https://github.com/great-expectations/great_expectations/pull/2124/

I'd opt to close my PR and focus on the that one. wdyt?

Reply:
> @alexsherstinsky This sounds reasonable, but I noticed that there's a nearly-identical PR from @EricSteg that's way more complete (e.g., includes updates to the docs)
> #2124
> 
> I'd opt to close my PR and focus on the that one. wdyt?

Yes, I like the changes in #2124 -- they look complete and exactly in line with my comment above.  Thank you for pointing me to it, and have a great weekend!